pom.xml

```diff
@@ -114,6 +114,7 @@
     <zeppelin.version>0.9.0-SNAPSHOT</zeppelin.version>
     <jgit.version>5.5.1.201910021850-r</jgit.version>
     <atomix.version>3.0.0-rc4</atomix.version>
+    <hive-metastore.version>2.1.1</hive-metastore.version>
```

Comment:
I think, need change `hive-metastore.version` to `hive.version`.

Reply:
Done.